pkgs/applications/science/biology/vcftools/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, htslib, zlib, autoreconfHook, pkgconfig, perl }:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "vcftools";
+  version = "0.1.15";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "vcftools";
+    rev = "v${version}";
+    sha256 = "15yxr4kidqb42gkbd6rjra6b07wpl6rgivlh9q73yavh5myafqk4";
+  };
+
+  buildInputs = [ autoreconfHook pkgconfig zlib perl ];
```

Comment:
```diff
- buildInputs = [ autoreconfHook pkgconfig zlib perl ];
+ buildInputs = [ zlib perl ];
+ nativeBuildInputs = [ autoreconfHook pkgconfig ];
```

Reply:
Cool - I found the GitHub issue describing the difference, but I'm not clear on how I would know which input goes where